tomls/oracles/pyth-snx.toml

```
@@ -7,7 +7,7 @@ target = ["system.oracle_manager.Proxy"]
 func = "registerNode"
 args = [
     5, # 5 = pyth aggregator type
-    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], [imports.pyth_erc7412_wrapper.contracts.PythERC7412Wrapper.address, settings.pythSnxFeedId, false]) %>",
+    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], ['0xEbe57e8045F2F230872523bbff7374986E45C486', settings.pythSnxFeedId, false]) %>",
```

need to revert this change
```suggestion
    "<%= defaultAbiCoder.encode(['address', 'bytes32', 'bool'], [imports.pyth_erc7412_wrapper.contracts.PythERC7412Wrapper.address, settings.pythSnxFeedId, false]) %>",
```

---

We need to do more troubleshooting on why it wasn't working per your suggestion before.

---

I was present at the deployment. This was a "try" but fid not fix issue. And was not reverted.